HelpSource/Classes/BelaScope.schelp

```diff
@@ -16,10 +16,20 @@ list::
 ## an instance of Server running on a Bela (either locally or remotely). 
 
 ## ServerOptions.belaMaxScopeChannels should be set > 0 for Bela's server. 
-It represents the maximum number of channels that is possible to scope on that server, and causes bus and resources allocation. 
+It represents the maximum number of channels that is possible to scope on that server, and affects resources allocation. 
 If set <= 0, the server won't instantiate any Scope object, and any attempt to instantiate a BelaScope from sclang will throw an error.
 ::
 
+note:: 
+When controlling a remote Bela server, it is necessary to manually set its code::belaMaxScopeChannels:: to the corresponding value set on the Bela when the server was booted:
```

Comment:
Would it be best/possible to query the server about its settings, instead of second-guessing it ?

Reply:
Not AFAIK... I'll ask around

Reply:
I think querying the server for its capabilities would also help for #82